environments/includes/elasticsearch.base.yml

```diff
@@ -15,5 +15,14 @@ services:
     volumes:
       - esdata:/usr/share/elasticsearch/data
 
+  elasticsearch-hq:
+    hostname: "${WARDEN_ENV_NAME}-elasticsearch-hq"
+    image: elastichq/elasticsearch-hq:latest
+    labels:
+      - traefik.enable=true
+      - traefik.http.routers.${WARDEN_ENV_NAME}-elasticsearch.tls=true
+      - traefik.http.routers.${WARDEN_ENV_NAME}-elasticsearch.rule=Host(`elastichq.${TRAEFIK_DOMAIN}`)
+      - traefik.http.services.${WARDEN_ENV_NAME}-elasticsearch.loadbalancer.server.port=5000
```

Comment:
Need to change `${WARDEN_ENV_NAME}-elasticsearch` to `${WARDEN_ENV_NAME}-elasticsearch-hq` so it won't conflict with rules on the `elasticsearch` container.

Reply:
@davidalger Thanks for review. I forgot to change it when reverted elasticsearch config.
Currently it's fixed.